### ISSUE TITLE
[ASK] Where is the artifact model downloaded by load_state_dict?

### ISSUE THREAD

**Issue by Clayrisee** [mlflow/mlflow#5773]
Hello MLFlow Team! 

I have a question, I already stored my state_dict artifact in S3 bucket and I want to load the artifact model using this code.

```python
pretrained_state_dict = mlflow.pytorch.load_state_dict(pretrained_url)
```

My question is, where is the downloaded ```state_dict.pth```  in my local computer?

Thank you

**Comment by dbczumar** [comment]
Hi @Clayrisee - great question! The state dict is downloaded to a temporary location on the filesystem that is not clearly exposed by this method. To download the state dict artifact to your local filesystem and obtain the resulting path, I recommend using the `mlflow.artifacts.download_artifacts` API (https://mlflow.org/docs/latest/python_api/mlflow.artifacts.html#mlflow.artifacts.download_artifacts), which is available in MLflow >= 1.25.0. For example:

```
pretrained_state_dict_path = mlflow.artifacts.download_artifacts(pretrained_url)
```

Thank you for using MLflow!

**Comment by Clayrisee** [comment]
> Hi @Clayrisee - great question! The state dict is downloaded to a temporary location on the filesystem that is not clearly exposed by this method. To download the state dict artifact to your local filesystem and obtain the resulting path, I recommend using the `mlflow.artifacts.download_artifacts` API (https://mlflow.org/docs/latest/python_api/mlflow.artifacts.html#mlflow.artifacts.download_artifacts), which is available in MLflow >= 1.25.0. For example:
> 
> ```
> pretrained_state_dict_path = mlflow.artifacts.download_artifacts(pretrained_url)
> ```
> 
> Thank you for using MLflow!

Thank you for your reply! its really great help for me!
